ci/build-jenkins.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+git submodule update --init --recursive
+cmake -DCMAKE_BUILD_TYPE=Release -H. -Bbuild-output
+cmake --build build-output -- -j $(nproc)
+
+cd build-output
+ctest --output-on-failure
+
+case ${BENCHMARK} in
+    endurance)
+        bigtable/benchmarks/endurance_benchmark ${PROJECT_ID} ${INSTANCE_ID} 4 82800;
+        ;;
+    latency)
+        bigtable/benchmarks/apply_read_latency_benchmark ${PROJECT_ID} ${INSTANCE_ID};
+        ;;
+    throughput)
```

Comment:
Can you please add a comment here about why 60 is used here?  Specifically that our tests run on a 4 CPU machine and that 15 threads per CPU seems to saturate 1 CPU.

Reply:
Will do.  We should talk about the actual number of threads though, my experiments required more like 128 threads to get 80% CPU utilization on a n1-standard-4 instance.

Reply:
And let's discuss this and fine tune the number before I merge the code.

Reply:
Just to keep you informed: after growing the cluster to 6 nodes, and if we include the upgrade to gRPC-v1.10.x (#283, it is pending review) I managed to get the CPU to 100% (or close, 99.x%) with 90 threads, with 60 I am getting ~89%, trying 75 threads right now to see how that does.